src/pages/projectSettings/tabs/ProjectTriggersTab/getFormSchema.tsx

```diff
@@ -175,3 +183,13 @@ export const getFormSchema = (
     },
   },
 });
+
+const LevelDescription = (
+  <>
+    Accepted values are task, build, and push. Task and build levels will
```

Comment:
I think you can drop the "Accepted values" sentence since that's implied by the dropdown, and then there's room to say a sentence about what Push is for